src/libstd/lib.rs

```diff
@@ -422,6 +423,11 @@ pub use core_collections::vec;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use std_unicode::char;
 
+// Reexport the start module on platforms that provide it
+#[unstable(feature = "start_fn", issue="0")]
+#[cfg(target_os = "redox")]
+pub use sys::start::*;
```

Comment:
I believe you just need the symbols exported, right? In that case they just need to be reachable publicly, not literally at the crate root. That's what the `rt` module suffices to do (it's entirely unstable)